src/components/Icon/DefaultAvatars.js

```diff
@@ -0,0 +1,51 @@
+import Avatar1 from '../../../assets/images/user-avatars/default-avatar_1.svg';
```

Comment:
I think I will rename this folder to something like `.../assets/images/avatars/user/`